src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const getDeclarationAsStrings = sourceString
+    .split(';')
+    .map((rawDeclaration) => rawDeclaration.trim());
+  const getArrayOfDeclarationEntries = getDeclarationAsStrings.map(
+    (preparedDeclaration) =>
+      preparedDeclaration
+        .split(':')
+        .map((preparedDeclarationToTrim) => preparedDeclarationToTrim.trim()),
+  );
```

Comment:
Similar to the previous comment, `getArrayOfDeclarationEntries` also implies a function but is an array. A name like `declarationEntries` would be more appropriate.